trade-finance-manager-ui/server/routes/case/cancellation.ts

```diff
@@ -0,0 +1,11 @@
+import { Router } from 'express';
+import { TEAM_IDS } from '@ukef/dtfs2-common';
+import { getReasonForCancelling } from '../../controllers/case/cancellation/reason-for-cancelling.controller';
+import { validateUserTeam } from '../../middleware';
+import { validateDealCancellationEnabled } from '../../middleware/feature-flags/deal-cancellation';
+
+export const cancellationRouter = Router();
+
+cancellationRouter.use(validateDealCancellationEnabled, validateUserTeam([TEAM_IDS.PIM]));
```

Comment:
I want to add some api tests for this stuff but there's no REDIS mocking/insertions set up. 

I wrote some tests but they were always redirecting to `/` because a session cookie needs to be added. I took a look at portal & gef-ui to see how they handle it. I wont add to this PR because it looks to be a bit of a rabbit-hole but this would be great, especially since we're adding a lot of endpoints in the near future